ocfweb/api/urls.py

```diff
@@ -15,4 +16,6 @@
     path('lab/desktops', lab.desktop_usage, name='desktop_usage'),
     path('session/log', session_tracking.log_session, name='log_session'),
     path('shorturl/<path:slug>', shorturls.bounce_shorturl, name='bounce_shorturl'),
+    path('stats/num_users', stats.get_num_users_in_lab, name='get_num_users_in_lab'),
+    path('stats/staff', stats.get_staff_in_lab, name='get_staff_in_lab'),
```

Comment:
imo lets make these make more sense, like `stats/lab/num_users` and `stats/lab/current_staff`. `stats/num_users` sounds like the total number of accounts or something

Reply:
Agreed, they'd fit well under the `lab` path that `lab/desktops` is already using, maybe `lab/stats/num_users`/`lab/stats/current_staff` instead of `stats/lab`?

Reply:
I actually originally had it as `lab/num_users`, I'll just revert it to that.